pkg/globalmanager/controllers/registry.go

```diff
@@ -14,19 +31,12 @@ type FeatureFactory = func(*runtime.ControllerContext) (runtime.FeatureControlle
 
 type Registry map[string]FeatureFactory
 
-func (r Registry) Register(name string, factory FeatureFactory) error {
-	if _, ok := r[name]; ok {
-		return fmt.Errorf("a feature controller named %s already exists", name)
-	}
-	r[name] = factory
-	return nil
-}
-
 func NewRegistry() Registry {
 	return Registry{
-		ji.Name: ji.New,
-		fl.Name: fl.New,
-		il.Name: il.New,
-		ll.Name: ll.New,
+		ji.Name:      ji.New,
```

Comment:
I don't think the name way is good way.

Reply:
well, these short alias names are only used in registry.go which is also small, I think it's ok here.